content/hardware-failure.md

```diff
@@ -156,7 +156,7 @@ make
 And now we can run it like so (this example will run it for 60 minutes/1 hour):
 
 ```bash
-./gpu_burn -d 3600
+./gpu_burn -tc 3600
```

Comment:
How does this behave on old GPUs without tensor cores?

Reply:
This will crash on cards older than RTX 20X0

Reply:
Let's keep both commands then, and explain that `-d` must be used for older GPUs.

Reply:
I removed the double switch for GTX GPUs and added another prompt for RTX GPUs with the -tc switch.

Reply:
![Screenshot from 2024-03-21 17-19-32](https://github.com/system76/docs/assets/78972098/017fc69e-55bf-4520-9f7d-647c403a7a9b)
![Screenshot from 2024-03-21 17-21-19](https://github.com/system76/docs/assets/78972098/a1a22b72-8593-425a-9896-4bd1e10d7428)
![Screenshot from 2024-03-21 17-23-01](https://github.com/system76/docs/assets/78972098/50caed4b-8b6e-43c0-919a-24cb34f7d598)
![Screenshot from 2024-03-21 17-25-00](https://github.com/system76/docs/assets/78972098/3721c3d4-c7be-4060-bc22-c9547674edda)


Just to add my 2¢ 
It is probably best to go with a multifaceted approach and do more variations for less time.
-d uses less power than using no argument other than time. While this may stress parts of the GPU more, it doesn't stress the power delivery components as much which could be where the failure is. 

./gpu_burn  1800
./gpu_burn -d 1800
./gpu_burn -tc 1800
./gpu_burn -d -tc 1800